src/procedures/__tests__/TransferErc20.ts

```diff
@@ -150,9 +171,10 @@ describe('TransferErc20', () => {
       );
     });
 
-    test('should add a transaction to the queue to transfer an erc20 token getting poly token funds if there are not enough funds, only on testnet', async () => {
+    test('should add an extra transaction to get POLY from the faucet if the balance is insufficient, specifically on testnet', async () => {
       wrappersMock.mock('isTestnet', Promise.resolve(true));
-      polyTokenMock.mock('address', Promise.resolve(params.tokenAddress));
+      polyTokenMock.mock('address', Promise.resolve(polyTokenAddress));
```

Comment:
Isn't this redundant since it is already being done in the `beforeEach` callback?

Reply:
Indeed